packages/x-components/src/utils/types.ts

```diff
@@ -197,3 +197,21 @@ export interface DebounceOptions {
 export interface ThrottleFunction<Params extends any[]> {
   (...args: Params): void;
 }
+
+/**
+ * Custom interface to provide a slot name to a Filter.
+ *
+ * @public
+ */
+export interface RenderFilter extends Filter {
```

Comment:
I don't think this is the best place to have this interface. Also, unless we need them, they should be internal. I would just add them in the components you use them

Reply:
Like we said, now I don't see any reason to not wrap the filter and facet into an object, instead of extending from them.